tests/test_traces.py

```diff
@@ -87,7 +87,7 @@ def test_Y():
         assert True
     """)
 
-    assert_that(report, has_error(message='xpassed',
+    assert_that(report, has_error(message='passed',
```

Comment:
That bothers me. Is there any info in pytest's changelog as to why they have changed this?